.github/workflows/release.yaml

```diff
@@ -15,12 +15,10 @@ on:
 
 jobs:
   create-github-release-push-tag:
-    uses: bakdata/ci-templates/.github/workflows/python-poetry-release.yaml@1.40.4
+    uses: bakdata/ci-templates/.github/workflows/bump-version-release.yaml@1.51.0
     name: Release
     with:
       release-type: ${{ inputs.release-type }}
-      python-version: "3.11"
-      poetry-version: "1.7.1"
       changelog: true
       changelog-config: "./.github/changelog-config.json"
```

Comment:
This can be removed now since the changelog is generated with [Git Cliff](https://github.com/orhun/git-cliff) now. I guess we forgot to remove [the input in the workflow](https://github.com/bakdata/ci-templates/blob/1.51.0/.github/workflows/bump-version-release.yaml#L21-L25), but it is never being used.